### DISCUSSION TITLE
Name must be unique?

### DISCUSSION THREAD

**Discussion by ghost** [community/community#27462] [Projects and Issues]
I have a yml defined at ISSUE_TEMPLATE folder:

```ymal
name: 🚀 Feature Request
description: I have a suggestion (and may want to implement it 🙂)!
labels: [feature request]
body:
  - type: markdown
    attributes:
      value: |
        You have an idea how to improve the site? That's awesome!
        Before submitting, please have a look at the existing issues if there's already
        something related to your suggestion.
  - type: textarea
    attributes:
      label: "Enter your suggestions in details:"
      placeholder: |
        1. What I expect to happen.
        2. Your reason.
        3. If possible, images or videos are welcome.
    validations:
      required: true
```
But the question is it tells me "Name must be unique"... So how to fix it?
![1659006416851](https://user-images.githubusercontent.com/52018749/181491075-66da1b22-c17a-4984-9403-846da2a60196.png)

**Comment by soumyamahunt** [comment]
Do you have issue template already with the same name? If not try removing the emoji if that succeds.

**Reply by AnonymousWP** [reply]
Doesn't help, there's no issue template with the same name in that directory.

**Comment by AnonymousWP** [comment]
I have a similar issue:

There's no other template with the same name. I double-checked that. If I use "Bug Report", the name must be unique error is absent. However, once I use "Bug report", it says the name must be unique.

**Comment by AnonymousWP** [comment]
I created a discussion myself now: https://github.com/orgs/community/discussions/110091
